libraries/AP_GPS/AP_GPS.cpp

```diff
@@ -2163,6 +2168,47 @@ bool AP_GPS::gps_yaw_deg(uint8_t instance, float &yaw_deg, float &accuracy_deg,
     return true;
 }
 
+#if GPS_MOVING_BASELINE && !defined(HAL_BUILD_AP_PERIPH)
+void AP_GPS::send_mavlink_RTCM(const uint8_t *data, uint16_t len)
+{
+    // Based on Mission Planner implmentation
+    // https://github.com/ArduPilot/MissionPlanner/blob/f64a63ff440b68987be91acd892ca55ba84d83e8/ExtLibs/ArduPilot/Mavlink/MAVLinkInterface.cs#L3637
+
+    uint8_t packets = (len / MAVLINK_MSG_GPS_RTCM_DATA_FIELD_DATA_LEN) + 1;
+    if ((len % MAVLINK_MSG_GPS_RTCM_DATA_FIELD_DATA_LEN) == 0) {
+        packets -= 1;
+    }
+
+    uint16_t offest = 0;
+    for (uint8_t i = 0; i<packets; i++) {
```

Comment:
If packets is > 4 you will need to either discard the data, or handle restarting the fragmenting over.